yamale/schema/schema.py

```diff
@@ -13,44 +12,48 @@ class Schema(object):
     Makes a Schema object from a schema dict.
     Still acts like a dict.
     """
-    def __init__(self, schema_dict, name='', validators=None):
+    def __init__(self, schema_dict, name='', validators=None, includes=None):
```

Comment:
I changed it so that included schemas also store the include dictionary in the object instead of passing the include data around since the argument lists where getting a bit long